exportReport/PanelSource.java

```diff
@@ -0,0 +1,214 @@
+ */
+
+/* 
+ * MODIFED BY	:	<NAME> - <GIT USER>
+ * MOD DATE		:	
+ * MOD VERSION	:	<VERSION OF PLUGIN>
+ * MOD DESC		:	
+ */
+
+public class PanelSource extends JPanel
+{
+	private JTextField			txtTitle		= null;
+	private JFormattedTextField	txtScanDate		= null;
+	private JTextField			txtReportDate	= null;
+	private JTextField			txtScanVer		= null;
+	private JTextField			txtReportVer	= null;
+	private JTextArea			txtDescription	= null;
+	private JTextField			txtBy			= null;
+	private JTextField			txtFor			= null;
+	ExtensionExportReport		extension		= null;
+
+	public PanelSource(ExtensionExportReport extension)
+	{
+		this.extension = extension;
+		initialize();
+	}
+
+	private void initialize()
+	{
+		String strLabel = Constant.messages.getString("exportReport.menu.source");
+		JLabel lblLabel = null;
+
+		String strTitle = Constant.messages.getString("exportReport.source.title");
+		JLabel lblTitle = null;
+		String ttTitle = Constant.messages.getString("exportReport.source.title.tooltip");
+
+		String strBy = Constant.messages.getString("exportReport.source.by");
+		JLabel lblBy = null;
+		String ttBy = Constant.messages.getString("exportReport.source.by.tooltip");
+
+		String strFor = Constant.messages.getString("exportReport.source.for");
+		JLabel lblFor = null;
+		String ttFor = Constant.messages.getString("exportReport.source.for.tooltip");
+
+		String strScanDate = Constant.messages.getString("exportReport.source.scandate");
+		JLabel lblScanDate = null;
+		String ttScanDate = Constant.messages.getString("exportReport.source.scandate.tooltip");
+
+		String strReportDate = Constant.messages.getString("exportReport.source.reportdate");
+		JLabel lblReportDate = null;
+		String ttReportDate = Constant.messages.getString("exportReport.source.reportdate.tooltip");
+
+		String strScanVer = Constant.messages.getString("exportReport.source.scanver");
+		JLabel lblScanVer = null;
+		String ttScanVer = Constant.messages.getString("exportReport.source.scanver.tooltip");
+
+		String strReportVer = Constant.messages.getString("exportReport.source.reportver");
+		JLabel lblReportVer = null;
+		String ttReportVer = Constant.messages.getString("exportReport.source.reportver.tooltip");
+
+		String strDescription = Constant.messages.getString("exportReport.source.description");
+		JLabel lblDescription = null;
+		String ttDescription = Constant.messages.getString("exportReport.source.description.tooltip");
+
+		JPanel top = null;
+		JPanel container = null;
+		JPanel content = null;
+		SpringLayout sl = null;
+
+		top = new JPanel(new FlowLayout(FlowLayout.LEFT));
+		container = new JPanel();
+		content = new JPanel();
+
+		this.setLayout(new BorderLayout());
+		this.add(top, BorderLayout.PAGE_START);
+		lblLabel = SharedFunctions.createLabel(top, lblLabel, strLabel, SharedFunctions.getTitleFont());
+
+		int[] pad =
+		{
+				0, 0, 295, 360
+		};
```

Comment:
It's just how my eclipse is set to format my text for readability. If it bothers you, i can change it.